hector_pose_prediction_ros/include/hector_pose_prediction_ros/visualization.h

```diff
@@ -0,0 +1,226 @@
+#ifndef HECTOR_POSE_PREDICTION_ROS_VISUALIZATION_H
+#define HECTOR_POSE_PREDICTION_ROS_VISUALIZATION_H
+
+#include <eigen_conversions/eigen_msg.h>
+#include <hector_math/types.h>
+#include <hector_pose_prediction_interface/types.h>
+#include <std_msgs/ColorRGBA.h>
+#include <visualization_msgs/MarkerArray.h>
+
+namespace hector_pose_prediction_ros
```

Comment:
For these ros extension packages just stay with the normal namespace `hector_pose_prediction_interface`.
Maybe the package name should also still include the interface part but I don't have a strong opinion on that.

Reply:
I think it could be irritating if the namespace does not match the package name.

Reply:
Since this is just splitting the ros functionality from the base package in order to separate the ros dependency, this is kind of part of the same package, so I would find it more irritating if I had to include a separate namespace.